pkg/3scale/amp/product/release_2_5.go

```diff
@@ -0,0 +1,47 @@
+package product
+
+type release_2_5 struct{}
+
+func (p *release_2_5) GetApicastImage() string {
+	return "registry.access.redhat.com/3scale-amp25/apicast-gateway"
+}
+
+func (p *release_2_5) GetBackendImage() string {
+	return "registry.access.redhat.com/3scale-amp25/backend"
+}
+
+func (p *release_2_5) GetBackendRedisImage() string {
+	return "registry.access.redhat.com/rhscl/redis-32-rhel7:3.2"
+}
+
+func (p *release_2_5) GetSystemImage() string {
+	return "registry.access.redhat.com/3scale-amp25/system"
+}
+
+func (p *release_2_5) GetSystemRedisImage() string {
+	return "registry.access.redhat.com/rhscl/redis-32-rhel7:3.2"
+}
+
+func (p *release_2_5) GetSystemMySQLImage() string {
+	return "registry.access.redhat.com/rhscl/mysql-57-rhel7:5.7"
+}
+
+func (p *release_2_5) GetSystemPostgreSQLImage() string {
+	return "registry.access.redhat.com/rhscl/postgresql-95-rhel7:9.5"
+}
+
+func (p *release_2_5) GetSystemMemcachedImage() string {
+	return "registry.access.redhat.com/3scale-amp20/memcached"
+}
+
+func (p *release_2_5) GetWildcardRouterImage() string {
+	return "registry.access.redhat.com/3scale-amp22/wildcard-router"
```

Comment:
I'm assuming no changes will be produced in WildcardRouter (I've seen there's no recent activity on it) so I guess we'll still keep using the same image than the one that was used in 2.4, which was 3scale-amp22.
Could you confirm that this would be the correct image to be used for the 2.5 release @mikz ?

Reply:
Yes, that should be the correct image.

Reply:
:+1: thank you for the confirmation